src/40_Dynamic_AMP/How_to_embed_interactive_elements_on_AMP_pages%3F.html

```diff
@@ -7,7 +7,10 @@
 
   This samples demonstrates how to embed interactive charts into AMP files.
 
-  To achieve its performance, AMP HTML doesn't allow custom Javascript. This doesn't mean that you can't created visually rich and interactive websites with AMP.
+  To achieve its performance, AMP HTML doesn't allow custom Javascript.
+  This doesn't mean that you can't created visually rich and interactive websites with AMP: [amp-bind](https://www.ampproject.org/docs/reference/components/amp-bind)
+  is the AMP component that allows to implement user interactivity on AMP pages but in case you are using
```

Comment:
Suggestion: rephrase to "This doesn't mean that you can't created visually rich and interactive websites with AMP. For example, you can use [amp-bind](/components/amp-bind)  to create dynamic and interactive features directly in AMP. However, sometimes dynamic content requires custom JS, e.g. when using a charting library. In this cases it's best to embed dynamic content using amp-iframe."